src/DurableTask.SqlServer/SqlOrchestrationService.cs

```diff
@@ -118,7 +118,7 @@ public override Task DeleteAsync(bool deleteInstanceStore)
             do
             {
                 using SqlConnection connection = await this.GetAndOpenConnectionAsync(cancellationToken);
-                using SqlCommand command = this.GetSprocCommand(connection, "dt._LockNextOrchestration");
+                using SqlCommand command = this.GetSprocCommand(connection, $"{this.settings.SchemaName}._LockNextOrchestration");
 
```

Comment:
Let's use a method for concatenation here, rather than every single line doing a string interpolation. We can also convert it to an extension method, so it can look like _LockNextOrchestration.PrependSchemaName()

Reply:
Added a new extension method to replace the interpolations and replaced all occurrences with the "PrependSchema()" method.

Reply:
Reverted the changes to use string interpolation again